.github/actions/json-args/action.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Pass a jsonified-list of files as shell arguments'
+description: 'Because file names with spaces are the worst'
+inputs:
+  list:
+    description: A jsonified Array of string file names
+    required: true
+  run:
+    description: "a command to run, where the literal '{}' will be replaced with the files as individual arguments. If no '{}' is provided, the files will be appended."
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/github-script@v6
+      with:
+        script: |
+          const listRaw = `${{ inputs.list }}`;
+          const files = JSON.parse(listRaw);
+          const {execSync} = require('child_process');
+          const filesList = files.map(name => `"${name}"`).join(' ')
```

Comment:
Should we do `${name.replace('"', '\"')}` here to protect against double-quotes in the filenames? (sorry, escaping is the wurst too).